src/organisms/cards/ContactCard/contact_card.module.scss

```diff
@@ -0,0 +1,37 @@
+@import 'src/assets/stylesheets/global';
+
+$contact-card-border: solid rem-calc(1px) $neutral-5;
+$contact-card-link-color: $neutral-2;
+
+.contact_card {
+  // display: none;
+  width: 100%;
+  padding: 0;
+  overflow: hidden;
+
+  font-family: $font-family-sans-serif;
+  font-size: rem-calc(13);
+  font-weight: bold;
+  text-align: left;
+  border: $contact-card-border;
+  border-radius: rem-calc(5px);
+
+  li + li {
+    border-top: $contact-card-border;
+  }
+
+  a {
+    display: block;
+    height: rem-calc(30px);
+    padding-left: rem-calc(20px);
```

Comment:
we like to use `ru` for padding/margin values. 1 ru (rhythm unit) = 1.5rem = 24px